shared/src/XLinkDispatcherImpl.c

```diff
@@ -176,6 +176,13 @@ int dispatcherLocalEventGetResponse(xLinkEvent_t* event, xLinkEvent_t* response)
         }
         case XLINK_CREATE_STREAM_REQ:
         {
+
+            // If Host side - this event happens when host tries to send an OpenStream request to device.
+            // Generate a streamId (and not relly on device to do it) and send that to the device.
```

Comment:
typo: rely

Reply:
Thanks - was wondering that this looks a tad odd:D
Fixing..